mulighetsrommet-arena-adapter/src/main/resources/application.yaml

```diff
@@ -3,14 +3,20 @@ server:
   host: 0.0.0.0
 
 app:
-  endpoints:
-    mulighetsrommetApi: ${MR_API_URI:-http://localhost:8080}
+  services:
+    mulighetsrommetApi:
+      url: ${MR_API_URL:-http://localhost:8080}
+      scope: ${MR_API_SCOPE:-default}
+
   database:
     host: ${DB_HOST:-localhost}
     port: ${DB_PORT:-5443}
     name: ${DB_DATABASE:-mulighetsrommet-arena-adapter-db}
     user: ${DB_USERNAME:-valp}
     password: ${DB_PASSWORD:-valp}
+
+  startKafkaTopicConsumption: false
```

Comment:
Kunne vi heller kalt denne for noe mer i duren `enable` isteden for `start`? Når det står start så kan det mistolkes som i at den aldri har kjørt noen gang hvis den settes tilbake til `false` etter en runde med `true`.

Og noen grunn til hvorfor denne ikke kunne vært under `kafka` hierarkiet i configen?